back-end/src/routes/api.js

```diff
@@ -24,6 +24,7 @@ const uploadFile = (request, response) => {
   (async () => {
     var statusCode = 400;
     var message = "File does not exist";
+    const {email, title, desc} = request.body;
     if (request.file) {
       // Read content from the file
       // const fileContent = fs.readFileSync(fileName);
```

Comment:
do you still need this

Reply:
Considering that you recommended it moments ago, who knows.